libs/httpproxy/proxy-camel-route/src/main/java/org/codice/proxy/http/HttpProxyServiceImpl.java

```diff
@@ -71,7 +72,8 @@
 
   private final Set<String> endpointIds = Collections.synchronizedSet(new HashSet<>());
 
-  public HttpProxyServiceImpl(CamelContext camelContext, String endpointType) {
+  public HttpProxyServiceImpl(CamelContext camelContext, String endpointType)
+      throws java.io.IOException {
```

Comment:
✏️ I think you can just put `throws IOException` here instead of giving the whole identifier